openverse_catalog/dags/maintenance/add_license_url.py

```diff
@@ -0,0 +1,179 @@
+"""
+# Add license URL
+
+Add license_url to all rows that have NULL in their meta_data fields.
+The license_url is constructed from the license and license_version fields.
+
+This is a maintenance DAG that should run once.
+"""
+import logging
+import os
+from datetime import datetime
+from textwrap import dedent
+from typing import Literal
+
+import jinja2
+from airflow.models import DAG
+from airflow.operators.python import BranchPythonOperator, PythonOperator
+from airflow.providers.postgres.hooks.postgres import PostgresHook
+from common import slack
+from common.constants import XCOM_PULL_TEMPLATE
+from common.licenses.constants import get_reverse_license_path_map
+from common.slack import send_message
+from psycopg2._json import Json
+
+
+DAG_ID = "add_license_url"
+DB_CONN_ID = os.getenv("OPENLEDGER_CONN_ID", "postgres_openledger_testing")
+
+ALERT_EMAIL_ADDRESSES = ""
+RETURN_ROW_COUNT = lambda c: c.rowcount  # noqa: E731
+
+logger = logging.getLogger(__name__)
+
+base_url = "https://creativecommons.org/"
+
+license_map = get_reverse_license_path_map()
```

Comment:
Since this is set up as a module-level variable, it'll be executed on each DAG parse cycle. It seems it's only used in the `update_license_url` function, perhaps we can just define it and call it there?

Reply:
Thank you for this suggestion and the explanation, I was counting on you all to improve this DAG :)